engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -137,22 +162,6 @@ namespace dmGameSystem
         uint8_t                             m_ReallocBuffers : 1;
     };
 
-    struct TexturesData
```

Comment:
Completely removed that structure because all information contains inside `SpriteComponent`